Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'haml-rails'
 gem 'jbuilder', '~> 2.5'
 gem 'mysql2'
 gem 'nokogiri', '>= 1.8.1'
+gem 'openssl'
```

Comment:
Just curious why this change

Reply:
Because it told me to? I actually cna't reproduce the problem anymore, but in Ruby 2.5, a number of the standard libraries were moved to "default gems". That is, they're gems, but gems that come with ruby. I've also had this problem crop up from time to time with `CSV`.

https://www.slideshare.net/hsbt/gemification-for-ruby-2530

Reply:
Ohhh okay.